js/modules/Kanban/Kanban.js

```diff
@@ -1552,7 +1552,7 @@ class GLPIKanbanRights {
         /**
        * Generate a user image based on the user's initials.
        * @since 9.5.0
-       * @param {string} teammember The teammember array/object that represents the user.
+       * @param {{}} teammember The teammember array/object that represents the user.
```

Comment:
Is that intended?

Reply:
Yes it was. The previous JSDoc type was a mistake and my IDE was giving an inspection error when trying to use the `replace` method on the `name` element.